exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterMetrics.java

```diff
@@ -87,9 +87,11 @@ private LongCounter exported() {
   }
 
   private Meter meter() {
-    return meterProviderSupplier
-        .get()
-        .get("io.opentelemetry.exporters." + exporterName + "-" + transportName);
+    MeterProvider meterProvider = meterProviderSupplier.get();
+    if (meterProvider == null) {
+      meterProvider = MeterProvider.noop();
+    }
```

Comment:
This is unrelated, but I noticed that we don't protect against `Supplier<MeterProvider>` returning null. This is one of the edge cases which nullaway doesn't catch.